transcom/mymove
# B-21488: Fixing UB Warning Message Display

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21488)
##[I-13640](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1044154)

## Summary
This is a fix for the work pushed up for B-21488.  Forgot to add check for UB Shipments for the UB Excess weight logic.

The UB shipment excess weight warning was showing up on moves that had no UB shipments. This was because there was no condition added to stop the UB logic from running when there is no UB Shipments. Simple fix. Add a check for UB shipments where if there are UB shipments then run the logic if not skip it.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmove and create a move without a UB shipment
2. Counsel the move
3. Approve the move as TOO
4. As a Prime sim update the estimated weight with a weight that will trigger an excess weight for the non UB shipment
5. Go view the Move as a TOO and you should not see the UB Excess weight message because there is no UB shipments


OPTIONAL(This was tested in the last PR and nothing changed the implementation)
7. Create a move with a UB shipment.
8. Update the estimated weight on the UB shipment to where it triggers and access weight
9. View the move as TOO. Should see the UB Shipment excess weight message

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
